src/DssSpell.sol

```diff
@@ -50,90 +90,222 @@ contract DssSpellAction is DssAction {
+
+        require(RwaInputConduitLike(RWA014_A_INPUT_CONDUIT_JAR).psm()                == MCD_PSM_USDC_A,                             "input-conduit-jar-psm-not-match");
+        require(RwaInputConduitLike(RWA014_A_INPUT_CONDUIT_JAR).to()                 == RWA014_A_JAR,                               "input-conduit-jar-to-not-match");
+        require(RwaInputConduitLike(RWA014_A_INPUT_CONDUIT_JAR).dai()                == DssExecLib.dai(),                           "input-conduit-jar-dai-not-match");
+        require(RwaInputConduitLike(RWA014_A_INPUT_CONDUIT_JAR).gem()                == DssExecLib.getChangelogAddress("USDC"),     "input-conduit-jar-gem-not-match");
+
+
+        // Init the RwaLiquidationOracle
+        RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).init(ilk, RWA014_A_INITIAL_PRICE, RWA014_DOC, RWA014_A_TAU);
+        (, address pip, , ) = RwaLiquidationLike(MIP21_LIQUIDATION_ORACLE).ilks(ilk);
+
+        // Init RWA014 in Vat
+        Initializable(MCD_VAT).init(ilk);
+        // Init RWA014 in Jug
+        Initializable(MCD_JUG).init(ilk);
+
+        // Allow RWA014 Join to modify Vat registry
+        DssExecLib.authorize(MCD_VAT, MCD_JOIN_RWA014_A);
 
-    address internal constant SPARK_D3M_PLAN            = 0x1fB2cF94D896bB50A17dD1Abd901172F088dF309;
-    address internal constant SPARK_D3M_POOL            = 0x8b6Ae79852bcae012CBc2244e4ef85c61BAeCE35;
-    address internal constant SPARK_D3M_ORACLE          = 0xa07C4eDB18E4B3cfB9B94D9CD348BbF6d5a7f4c2;
+        // 1m debt ceiling
+        DssExecLib.increaseIlkDebtCeiling(ilk, RWA014_A_LINE, /* _global = */ true);
 
-    address internal constant SPARK_ADAI                = 0x4480b29AB7a1b0761e6d0d480325de28B7266d73;
-    address internal constant SPARK_DAI_STABLE_DEBT     = 0xD72630D78157E1a2feD7A329873Bfd496704403D;
-    address internal constant SPARK_DAI_VARIABLE_DEBT   = 0xa99d874d26BdfD94d474Aa04f4f7861DCD55Cbf4;
-    address internal constant SPARK_POOL_CONFIGURATOR   = 0xe0C7ec61cC47e7c02b9B24F03f75C7BC406CCA98;
+        // Set price feed for RWA014
+        DssExecLib.setContract(MCD_SPOT, ilk, "pip", pip);
 
-    address internal constant WBTC                      = 0x91277b74a9d1Cc30fA0ff4927C287fe55E307D78;  // Please note this is not the same WBTC as in Maker
-    address internal constant INTEREST_RATE_STRATEGY    = 0x491acea4126E48e9A354b64869AE16b2f27BE333;
+        // Set collateralization ratio
+        DssExecLib.setIlkLiquidationRatio(ilk, RWA014_A_MAT);
 
-    uint256 internal constant MILLION = 10 ** 6;
-    uint256 internal constant WAD = 10 ** 18;
-    uint256 internal constant RAD = 10 ** 45;
+        // Poke the spotter to pull in a price
+        DssExecLib.updateCollateralPrice(ilk);
+
+        // Give the urn permissions on the join adapter
+        DssExecLib.authorize(MCD_JOIN_RWA014_A, RWA014_A_URN);
+
+        // MCD_PAUSE_PROXY and Monetalis permission on URN
```

Comment:
Are mentions of Monetalis accurate in this context? They were for RWA007 but is that still the case?

Reply:
Yes, it's accurate. Still use them in this deal